movement/watch_faces/complication/day_one_face.h

```diff
@@ -30,13 +30,22 @@
 // The Day One face is designed to count upwards from the wearer's date of birth. It also functions as an
```

Comment:
Could you change this to say that it counts the days since or until a given date?

Reply:
Done!